docs/alpha-ref.json

```diff
@@ -1,3 +0,0 @@
-{
-    "appref": "v"
```

Comment:
Are you sure you want to delete the reference? This will take down anyone pointing to /apha get 404.

Reply:
It is currently redundant to /beta and was only used internally. But we can leave it until we need to differentiate against /beta again. I have added it back.